README.md

```diff
@@ -137,6 +137,12 @@ Rules are grouped by category to help you understand their purpose. Each rule ha
 | :car: | [no-classic-classes](./docs/rules/no-classic-classes.md) | disallow "classic" classes in favor of native JS classes |
 | :white_check_mark::wrench: | [no-ember-super-in-es-classes](./docs/rules/no-ember-super-in-es-classes.md) | disallow use of `this._super` in ES class methods |
 
+### Ember Utils
+
+|    | Rule ID | Description |
+|:---|:--------|:------------|
+|  | [no-tryinvoke](./docs/rules/no-tryinvoke.md) | disallow usage of the Ember's `tryInvoke` util |
```

Comment:
Can we name it `no-try-invoke` similar to how we dasherize other functions in lint rule names like `no-get-with-default`?